src/roles/common/tasks/check-prereq.yml

```diff
@@ -19,34 +20,36 @@
       msg:  "Metro only supports RHEL/CentOS as an Ansible host"
 
   - name : Output installed packages yum
-    shell: yum list installed
+    command: yum list installed         # noqa 303
```

Comment:
Maybe not use command:

- name: List packages
  yum:
    list: installed
  register: yumoutput

Reply:
Yes, Sorry I somehow missed in the yum module examples that it has this option too. Thanks for pointing out